tools/cmake/mbed_greentea.cmake

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0# the macro
+
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_SOURCE_DIR}/.mbedbuild CACHE INTERNAL "")
+
+include(${MBED_PATH}/tools/cmake/app.cmake)
+
+# CMake Macro for generalizing CMake configuration across the greentea test suite with configurable parameters
+# Macro args:
+# TEST_NAME - Test suite name
+# TEST_INCLUDE_DIRS - Test suite include directories for the test
+# TEST_SOURCES - Test suite sources
+# TEST_REQUIRED_LIBS - Test suite required libraries
+# 
+# calling the macro:
+# mbed_greentea_cmake_macro(
+#    TEST_NAME mbed-platform-system-reset
+#    TEST_INCLUDE_DIRS mbed_store
+#    TEST_SOURCES foo.cpp bar.cpp
+#    TEST_REQUIRED_LIBS mbed-kvstore mbed-xyz
+# )
+
+macro(mbed_greentea_cmake_macro)
```

Comment:
it seems everything between `macro()` and `endmacro()` is usually indented for better readability.
https://cmake.org/cmake/help/latest/command/macro.html